### ISSUE TITLE
HyprPanel and tidal-hifi Lock up When Both are Running

### ISSUE THREAD

**Issue by awsumatt** [Jas-SinghFSU/HyprPanel#663]
**Describe the bug**
Hard lock of both the panel and Tidal Music app (tidal-hifi)

**To Reproduce**
Have hyprpanel running and launch tidal-hifi

**Expected behavior**
Tidal and HyprPanel wouldn't be mutually exclusive in functionality

**Desktop (please complete the following information):**
 - Arch Linux / Zen 6.12.6-1-1 
 - Hyprland

**Additional context**
I was hoping it would be resolved in agsv2 but unfortunately it is a problem that has persisted through both implementations. It doesn't seem to be an Electron issue, Discord works fine. Not sure what the cause is

**Comment by Jas-SinghFSU** [comment]
Yeah, I was able to replicate this with the Astal media player example too so it seems like an upstream issue. I'll try to dig in a create an issue on the Astal repo.

**Comment by awsumatt** [comment]
Your notification hunch seems to be correct. Disabling notifications seems to be a solid workaround for now
